### ISSUE TITLE
Non well formed numeric value encountered

### ISSUE THREAD

**Issue by AykutCevik** [mgtcommerce/Mgt_Developertoolbar#33]
After enabling the extension I'm getting this error:
``` 1 exception(s):
Exception #0 (Exception): Notice: A non well formed numeric value encountered in /home/cloudpanel/htdocs/magento.dev/vendor/mgtcommerce/module-mgtdevelopertoolbar/src/app/code/Mgt/DeveloperToolbar/Block/Toolbar/Block/MemoryConsumption.php on line 70

Exception #0 (Exception): Notice: A non well formed numeric value encountered in /home/cloudpanel/htdocs/magento.dev/vendor/mgtcommerce/module-mgtdevelopertoolbar/src/app/code/Mgt/DeveloperToolbar/Block/Toolbar/Block/MemoryConsumption.php on line 70
#0 /home/cloudpanel/htdocs/magento.dev/vendor/mgtcommerce/module-mgtdevelopertoolbar/src/app/code/Mgt/DeveloperToolbar/Block/Toolbar/Block/MemoryConsumption.php(70): Magento\Framework\App\ErrorHandler->handler(8, 'A non well form...', '/home/cloudpane...', 70, Array)
#1 /home/cloudpanel/htdocs/magento.dev/vendor/mgtcommerce/module-mgtdevelopertoolbar/src/app/code/Mgt/DeveloperToolbar/Block/Toolbar/Block/MemoryConsumption.php(77): Mgt\DeveloperToolbar\Block\Toolbar\Block\MemoryConsumption->Mgt\DeveloperToolbar\Block\Toolbar\Block\{closure}('768M')
#2 [internal function]: Mgt\DeveloperToolbar\Block\Toolbar\Block\MemoryConsumption->getPhpMemoryLimit() 
```

Running PHP7, should work on  PHP5 due to automatically casting.

**Comment by AykutCevik** [comment]
Should be fixed with #32 - when do you plan to merge it?

**Comment by swieczorek** [comment]
Fixed
